Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -621,6 +620,12 @@ def import_tests(self):
         for prefixed_value in sorted(total_prefixed_property_values, key=lambda p: total_prefixed_property_values[p]):
             _log.info('  %s: %s', prefixed_value, total_prefixed_property_values[prefixed_value])
 
+        # Fetch the Upstream commit ID for the WPT import So it is ready to insert in the commit message.
+        upstream_id = self.test_downloader().upstream_commitID_head
+        _log.info('\n--------- Please include the following in your commit message: ---------\n')
+        _log.info(f'Upstream commit: https://github.com/web-platform-tests/wpt/commit/{upstream_id}\n')
```

Comment:
Oh the code has to be compatible with Python 2?
@JonWBedard 

```
  File "/Volumes/Data/worker/WebKitPy-Tests-EWS/build/Tools/Scripts/webkitpy/w3c/test_importer.py", line 626
    _log.info(f'Upstream commit: https://github.com/web-platform-tests/wpt/commit/{upstream_id}\n')
                                                                                                 ^
SyntaxError: invalid syntax
```

Reply:
Yes, currently. We might at some point try to cut down having to support Python 2 throughout webkitpy, but it's potentially not worth the effort versus removing Python 2 support altogether.